std/net/curl.d

```diff
@@ -627,6 +627,60 @@ unittest
     assert(res == cast(ubyte[])[17, 27, 35, 41]);
 }
 
+/* Encode associative array using www-form-urlencoding
+ * perhaps this should live in std.uri
```

Comment:
yes it should ;-)
Please move and add `Params`, `Returns` and `See_Also`.

Quite many language libraries actually provide this to the user, so chances are we might want to expose this in the future, but for now just set it to `package` - adding a new public function requires approval.